x/ibc/02-client/types/tendermint/misbehaviour.go

```diff
@@ -1,96 +1,48 @@
 package tendermint
 
 import (
-	"fmt"
-
-	yaml "gopkg.in/yaml.v2"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	evidenceexported "github.com/cosmos/cosmos-sdk/x/evidence/exported"
 	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
 	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/types/errors"
-
-	"github.com/tendermint/tendermint/crypto/tmhash"
-	cmn "github.com/tendermint/tendermint/libs/common"
-	tmtypes "github.com/tendermint/tendermint/types"
+	host "github.com/cosmos/cosmos-sdk/x/ibc/24-host"
 )
 
-var _ exported.Evidence = Evidence{}
+var _ exported.Misbehaviour = Misbehaviour{}
+var _ evidenceexported.Evidence = Misbehaviour{}
 
-// Evidence is a wrapper over tendermint's DuplicateVoteEvidence
-// that implements Evidence interface expected by ICS-02
-type Evidence struct {
-	*tmtypes.DuplicateVoteEvidence
-	ChainID        string `json:"chain_id" yaml:"chain_id"`
-	ValidatorPower int64  `json:"val_power" yaml:"val_power"`
-	TotalPower     int64  `json:"total_power" yaml:"total_power"`
+// Misbehaviour contains an evidence that a
+type Misbehaviour struct {
+	*Evidence
```

Comment:
Why does Misbehaviour contain `ChainID` when Evidence already has `ChainID` field?

Reply:
`Misbehaviour` contains only the `ClientID`